yt/frontends/cholla/data_structures.py

```diff
@@ -157,6 +157,10 @@ def _is_valid(cls, filename, *args, **kwargs):
         except AttributeError:
             return False
         else:
-            return "bounds" in attrs and "domain" in attrs
+            # Avoiding issue where CHOLLA LightRay datasets get selected
+            if attrs["data_type"] == "yt_light_ray":
```

Comment:
What happens if `attrs` doesn't have `data_type`?  How about making this part of the below statement, with something like `return attrs.get("data_type", None) != "yt_light_ray"` or something?

Reply:
Good call!  I'm not thinking this morning.